e2e/test/iothub/command/CommandE2ETests.cs

```diff
@@ -39,27 +39,23 @@ public class CommandE2ETests : E2EMsTestBase
         private static readonly TimeSpan s_defaultCommandTimeoutMinutes = TimeSpan.FromMinutes(1);
 
         [LoggedTestMethod]
-        public async Task Command_DeviceReceivesCommandAndResponse_Mqtt()
+        [DataRow(Client.TransportType.Mqtt_Tcp_Only)]
+        [DataRow(Client.TransportType.Mqtt_WebSocket_Only)]
+        [DataRow(Client.TransportType.Amqp_Tcp_Only)]
+        [DataRow(Client.TransportType.Amqp_WebSocket_Only)]
```

Comment:
Tests are currently parallelized at class-level, so non-parallelization of parameterized tests isn't an issue right now. Instead of parallelizing at method-level, we should instead invest in migrating off MS test and to NUnit/xUnit.